src/eval.ml

```diff
@@ -143,17 +143,23 @@ and eq_modulo : term -> term -> bool = fun a b ->
   let res = try eq_modulo [(a,b)]; true with Exit -> false in
   if !log_enabled then log_conv (r_or_g res "%a == %a") pp a pp b; res
 
-(** {b NOTE} that matching with trees involves three collections of terms.
+(** {b NOTE} that matching with trees involves two collections of terms.
```

Comment:
Shouldn't this comment be inside the code of tree_walk, to comment the auxiliary function walk?

Reply:
It won't be visible in the generated doc, or will it be?

Reply:
I don't know. I am not expert in ocamldoc. But this is confusing to talk about things that are not arguments of the following function. Or else you speak of [walk] inside the comment for [tree_walk].

Reply:
Indeed it won't.